benchmarks/run_release_benchmark.py

```diff
@@ -0,0 +1,212 @@
+import boto3
+import click
+import os
+import paramiko
+import time
+
+
+def get_boto_client(service_name):
+    client = boto3.client(
+        service_name,
+        region_name='us-east-1',
+    )
+    return client
+
+
+def get_boto_resource(service_name):
+    client = boto3.resource(
+        service_name,
+        region_name='us-east-1',
+    )
+    return client
+
+
+def start_ec2_instance(key_name, ami, security_group, subnet_id, instance_type='cpu'):
+    print("Starting {} EC2 instance".format(instance_type))
+    ec2_client = get_boto_client('ec2')
+    instances = ec2_client.run_instances(
+        BlockDeviceMappings=[
+            {
+                'DeviceName': '/dev/sdh',
+                'Ebs': {
+                    'DeleteOnTermination': True,
+                    'VolumeSize': 200,
+                    'VolumeType': 'standard',
+                },
+            },
+        ],
+        ImageId=ami,
+        InstanceType='c4.4xlarge' if instance_type == "cpu" else "p3.8xlarge",
```

Comment:
Can this be a made an arg in preset config ?

Reply:
Preset config is used by the benchmark-ab.py script to run the benchmark. This script is just targetted at automating the release benchmarking process where we use the above two types of instances, hence hard-coded based on the instance type supplied to the script (Default: CPU).